samples/integration/cmake-vcpkg/README.md

```diff
@@ -0,0 +1,532 @@
+# Develop an Azure C++ App using vcpkg and CMake
+
+## Perquisite
+
+1. Any Text Editor
+2. A terminal
+3. A C++ compiler
+    - Windows: [MSVC](https://learn.microsoft.com/cpp/build/vscpp-step-0-installation?view=msvc-170)
+4. [git](https://git-scm.com/downloads)
+
+## Steps
+
+### Install CMake
+
+- Go to the [CMake Downloads page](https://cmake.org/download/) where you can either download the latest version of the CMake source or an installer for your OS. I'd recommend using the installer for simplicity. When asked select the option to add CMake to your PATH.
+- Once you've downloaded and run through the installer program run the following command to verify CMake has been installed properly:
+
+```bash
+cmake --version
+```
+
+- You should get an output denoting the currently install version of CMake. Like this:
+
+```bash
+cmake version 3.30.2
+```
+
+- If you don't see the expected output you may need to restart your computer.
+
+### Install vcpkg
+
+- To install vcpkg you'll first need to clone the vcpkg repo. I recommend cloning vcpkg to a central location on your development environment and not in your C++ Project directory. In this example I'll clone vcpkg to my home dir.
+
+```bash
+cd ~
+git clone https://github.com/microsoft/vcpkg.git
+```
+
+- Once the vcpkg repo is cloned traverse into the new directory and run the bootstrap-vcpkg script.
+  - If you're on Windows run:
+
+  ```bash
+  cd .\vcpkg\
+  .\bootstrap-vcpkg.bat
+  ```
+
+  - If you're on MacOS or Linux, run:
+  
+  ```bash
+  cd vcpkg
+  ./bootstrap-vcpkg.sh
+  ```
+
+- After bootstrapping vcpkg, add it to your path so you can access the vcpkg executable from your project directory. Remember to replace the path in the command example with the path to the vcpkg directory you cloned earlier.
+  - On Windows Powershell, enter:
+  
+  ```bash
+  $env:Path = "$env:Path;C:\path\to\vcpkg"
+  ```
+
+  - On MacOS or Linux systems, enter:
+  
+  ```bash
+  PATH=$PATH:/path/to/vcpkg
+  ```
+
+- Traverse back to your project directory and run the following to verify the vcpkg directory was added to your path:
+
+```bash
+vcpkg --version
+```
+
+- You should get an output resembling the following (The hash at the end of the line does not need to match exactly):
```

Comment:
The version will also almost certainly not match as well - this version works today but won't in a couple of months.